src/python/WMCore/MicroService/MSPileup/MSPileupData.py

```diff
@@ -241,9 +241,7 @@ def updatePileup(self, doc, rseList=None, validate=True, userDN=''):
         fraction = doc.get('containerFraction', 1.0)
```

Comment:
here it is not an issue since we go through validation when we write document to MongoDB, and during validation we create MSpileupObj which has defaults, in this case it user didn't provide container fraction it will be set to 1.

Reply:
does it mean we can safely delete this line then?

Reply:
no, we still need to get fraction from the doc. I can't just delete this line since `fraction` in the one provided in input document, and we compare it (few lines below) with fraction which we obtain from document in a database.

Reply:
Yep, sorry for the ill thought. I meant to have `fraction = doc['containerFraction']` to avoid any undesired use of the default value. But I guess it's not really needed and we might just mark this comment as resolved.